docs/user/dashboard/aggregation-reference.asciidoc

```diff
@@ -0,0 +1,222 @@
+[[aggregation-reference]]
+== Aggregation reference
+
+{kib} supports many types of {ref}/search-aggregations.html[{es} aggregations] that you can use to build complex summaries of your data. 
+
+By using a series of {es} aggregations to extract and process your data, you can create panels that tell a 
+story about the trends, patterns, and outliers in your data.
+
+[float]
+[[bucket-aggregations]]
+=== Bucket aggregations
+
+For information about Elasticsearch bucket aggregations, refer to {ref}/search-aggregations-bucket.html[Bucket aggregations].
+
+[options="header"]
+|===
+
+| Type | Visualizations | Data table | Markdown | Lens | TSVB 
+
+| Histogram
+^| X
+^| X
+^| X
+| 
+| 
+
+| Date histogram
+^| X
+^| X
+^| X
+^| X
+^| X
+
+| Date range
+^| X
+^| X
+^| X
+| 
+| 
+
+| Filter
```

Comment:
This aggregation is something we should discourage, I would consider removing it from the list.

Reply:
But if users see it in the UI, then go to the docs and don't see it there, won't that causes confusion?